openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -733,6 +735,13 @@ def _get_representations(self, instance, exp_files, do_not_add_review):
             if not already_there:
                 representations.append(rep)
 
+        for rep in representations:
+            # inject colorspace data
+            self.set_representation_colorspace(
+                rep, self.context,
+                colorspace=instance["colorspace"]
```

Comment:
Typo?
```suggestion
                colorspace=instance.data["colorspace"]
```

Reply:
It is actually `instance.data`, renamed to make it clearer. Got similar issue previously too.